src/canvas-extractor/canvas_extractor/__main__.py

```diff
@@ -0,0 +1,49 @@
+﻿# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+from typing import List
+from dotenv import load_dotenv
+from pandas import DataFrame
+import sqlalchemy
+
+from canvasapi import Canvas
+from canvasapi.assignment import Assignment
+from canvasapi.course import Course
+from canvasapi.enrollment import Enrollment
+from canvasapi.submission import Submission
+from canvasapi.user import User
+
+
+from api.assignments import assignments_synced_as_df, request_assignments
+from api.courses import courses_synced_as_df, request_courses
+from api.enrollments import enrollments_synced_as_df, request_enrollments
+from api.students import request_students, students_synced_as_df
+from api.submissions import request_submissions, submissions_synced_as_df
+from config import get_canvas_api, get_sync_db_engine
+# import export_data
+
+load_dotenv()
+
+canvas: Canvas = get_canvas_api()
+
```

Comment:
I prefer having the code inside functions, then calling them in `main()`. It could be separated by task(request, sync, export) or by resource(get_courses, get_students...)... In the case of separating by resource it could make all the calls for get, sync and export the resource.

Reply:
Yes, it's pretty simplistic, I'll update.  Best would be to align on resource.  Hadn't organized code here at the start of the work because I didn't know what the dependencies would look like.  Turns out it's simple if you maintain lists of the canvasapi result objects.

Reply:
i'll do this in my next PR